versions.mk

```diff
@@ -11,4 +11,4 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-GOLANG_VERSION ?= 1.23.5
+GOLANG_VERSION ?= 1.24.0
```

Comment:
Question: Could we use `hack/golang-version.sh` here instead?

Reply:
(This would mean that we have a golang Dockerfile in `deployments/devel`).

Reply:
sure, a way to leverage dependabot, do we agree that said image wouldn't require an `image` action, right?

Reply:
Done, we now have a `deployments/devel` structure

Reply:
I have removed this file in favor of the `deployments/devel` structure